packages/js/client-config-builder/src/bundles/getDefaultConfig.ts

```diff
@@ -15,48 +13,50 @@ import { fileSystemPlugin } from "@polywrap/fs-plugin-js";
 import { loggerPlugin } from "@polywrap/logger-plugin-js";
 import { fileSystemResolverPlugin } from "@polywrap/fs-resolver-plugin-js";
 import { concurrentPromisePlugin } from "concurrent-plugin-js";
+import { ExtendableUriResolver } from "@polywrap/uri-resolver-extensions-js";
+import path from "path";
+import { WasmPackage } from "@polywrap/wasm-js";
+import fs from "fs";
```

Comment:
This will cause problems in the browser. Instead we should be embedding the raw bytes of the files within a JS module so that reading from a file is not needed.